File: native-client/windows.m
#import <Foundation/Foundation.h>
#import <CoreGraphics/CoreGraphics.h>
int main(){@autoreleasepool{NSArray *ws=(__bridge_transfer NSArray*)CGWindowListCopyWindowInfo(kCGWindowListOptionAll|kCGWindowListExcludeDesktopElements,kCGNullWindowID);for(NSDictionary*w in ws){NSString*n=w[(id)kCGWindowOwnerName];if([n.lowercaseString containsString:@"java"]||[n.lowercaseString containsString:@"minecraft"])NSLog(@"%@",w);}}return 0;}

File: native-client/windows.swift
import Foundation
import CoreGraphics
let windows=CGWindowListCopyWindowInfo([.optionAll,.excludeDesktopElements], kCGNullWindowID) as? [[String:Any]] ?? []
for w in windows {let owner=w[kCGWindowOwnerName as String] as? String ?? "";if owner.lowercased().contains("java") || owner.lowercased().contains("minecraft"){print(w)}}
